app/js/components/orderEntry/ActiveOrders.jsx

```diff
@@ -82,8 +82,18 @@ export class ActiveOrders extends React.Component {
           </td>
           {details}
           <td className="text-center action-padding">
-            <a href="#"> <i className="icon-edit" title="Edit" /> </a>
-            <a href="#"> <i className="icon-remove" title="Delete" /> </a>
+            {
+              this.props.editDrugUuid ?
+                <p> Will REVISE </p> :
```

Comment:
Did you intentionally put REVISE in call caps?

Reply:
@dkayiwa Yes, that is how the old module had been implemented.

Reply:
Do you have a screenshot of the old module?

Reply:
<img width="965" alt="screen shot 2018-05-02 at 4 03 24 pm" src="https://user-images.githubusercontent.com/22788274/39524704-76383aee-4e22-11e8-82ac-76a8615fe612.png">